fix: make clear service endpoints are not mandatory (#241)
KILTprotocol/docs: docs/develop/02_chain/02_pallets/01_did.md

A full DID is needed if the user wants to become an Attester or wants to setup delegations.
A full DID also allows the user to embed a list of URLs, known as service endpoints, into the DID document so that they can be retrieved from the chain as part of the DID document.

> To create a full DID the user first has to create a set of keys, where some of them are optional and can be set later on:

Reviewer: What about 
```suggestion
To create a full DID the user first has to create some keys, and optionally some services
```

We should move away from calling services service endpoints, and just use services 😄.

* one authentication key for signing extrinsics from your DID
* zero or more key agreement keys for encrypting messages that are sent to you

> * one attestation key for signing attestations (optional)

Reviewer: Maybe we can move the `[OPTIONAL]` in front?

* one delegation key for authorizing delegations (optional)
* service endpoints that point to external hostings for others to find (optional)

> With those mandatory keys prepared, they are ready to write the DID to the KILT blockchain.

Reviewer: ```suggestion
After the relevant components have been created, they are ready to write the DID to the KILT blockchain.
```
